packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -430,7 +430,7 @@ class DatePicker extends React.Component {
         >
           <ReactDatePicker
             {...customProps}
-            withPortal={this.state.showPortalPicker}
+            withPortal={(customProps.inline) ? false : this.state.showPortalPicker}
```

Comment:
Should we make `inline` a first class prop but make it private?

Reply:
I add it as private prop but later than i thought of using it with customProps itself.